cityofaustin/atd-moped
# Updates `moped_tags` for Mobility Annual Plan (MAP)

## Associated issues
https://github.com/cityofaustin/atd-data-tech/issues/10362

## Testing
Local or[ test deployment](https://10362-jc-update-moped-tags--atd-moped-main.netlify.app/moped/session/signin)

**Steps to test:**
1. Navigate to `moped/dev/lookups#moped-tags`
2. Verify this is the complete list of tags:

Type | Name | Slug
-- | -- | --
Work Plan | MAP Signals 2023 | map_signals2023
Work Plan | MAP TIA Fiscal | map_tiafiscal
Work Plan | MAP Transit Enhancement 2023 | map_transitenhancement2023
Work Plan | MAP Transit Speed and Reliability (CapMetro ILA) 2023 | map-transit-speed-and-reliability-capmetro-ila-2023
Work Plan | MAP Vision Zero - Major Safety 2023 | map-vision-zero-major-safety-2023
Work Plan | MAP Vision Zero - Ped Safety 2023 | map-vision-zero-ped-safety-2023
Work Plan | MAP Vision Zero - Rapid Response/Small Scale 2023 | map-vision-zero-rapid-response-small-scale-2023
Work Plan | MAP Vision Zero - Speed Management 2023 | map_vzero_speedmanagement2023
Work Plan | MAP Vision Zero - Street Lighting 2023 | map-vision-zero-street-lighting-2023
Work Plan | MAP Vision Zero - Systemic Safety 2023 | map-vision-zero-systemic-safety-2023
Work Plan | MAP Vision Zero - Systemic Safety Curves 2023 | map_vzero_systemicsafety_curves2023



---
#### Ship list
